esptfa_arima/Authentication/views.py

```
@@ -46,35 +62,72 @@ def register(request):
+                )
+                response.set_cookie(
+                    key=settings.SIMPLE_JWT['AUTH_COOKIE_REFRESH'],
+                    value=str(refresh),
+                    max_age=settings.SIMPLE_JWT['REFRESH_TOKEN_LIFETIME'].total_seconds(),
+                    secure=settings.SIMPLE_JWT['AUTH_COOKIE_SECURE'],
+                    httponly=settings.SIMPLE_JWT['AUTH_COOKIE_HTTP_ONLY'],
+                    samesite=settings.SIMPLE_JWT['AUTH_COOKIE_SAMESITE'],
+                    path='/'
+                )
+                return response
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)